Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -390,19 +400,22 @@ def _create_search_args(self):
 
         return filters
 
+    @logger
     def _create_exposure_filters(self, exposure_types):
         """Determine exposure types based on alert type"""
-
+        exp_types = []
         if self._is_cloud_exfiltration:
-            exp_types = []
-            if self._PUBLIC_SEARCHABLE in exposure_types:
-                exp_types.append(ExposureType.IS_PUBLIC)
-            if self._PUBLIC_LINK in exposure_types:
-                exp_types.append(ExposureType.SHARED_VIA_LINK)
-            return [ExposureType.is_in(exp_types)]
+            for t in exposure_types:
+                exp_type = self.exposure_type_map.get(t)
+                if exp_type:
+                    exp_types.append(exp_type)
+                else:
+                    LOG("Received unsupported exposure type {0}.".format(t))
+            if exp_types:
+                return [ExposureType.is_in(exp_types)]
```

Comment:
if the `exp_types` is empty due to it being unsupported, this will result in an ffs query of _all_ events in the given time frame. If we returned `[ExposureType.not_in(exp_types)]`, this would actually return the result set for the events they were looking for.

Reply:
I don't understand.

Reply:
Why would return [ExposureType.not_in(exp_types)] is expt_types is empty?

Reply:
if we find an alert that was triggered by an unsupported type, doing `not_in(list_of_supported_types)` would return only the not supported ones